src/Mod/Sketcher/Gui/CommandCreateGeo.cpp

```diff
@@ -349,6 +446,7 @@ class DrawSketchHandlerLine: public DrawSketchHandler
             bool continuousMode = hGrp->GetBool("ContinuousCreationMode",true);
             if(continuousMode){
                 // This code enables the continuous creation mode.
+                onWidgetChanged();

```

Comment:
I get what you want to do, but the widget did not change. This misleads the one reading the code. The most important thing about code is to show intention. Here, we show the intention to notify that a widget changed, when it did not.

Here is best to do some code reuse and call a function that actually resets the widget.

Do not change at this moment.